requirements.txt

```diff
@@ -0,0 +1,62 @@
+#
+# This file is autogenerated by pip-compile with Python 3.8
+# by the following command:
+#
+#    pip-compile --generate-hashes --output-file=requirements.txt requirements.in
+#
+cachetools==5.4.0 \
+    --hash=sha256:3ae3b49a3d5e28a77a0be2b37dbcb89005058959cb2323858c2657c4a8cab474 \
+    --hash=sha256:b8adc2e7c07f105ced7bc56dbb6dfbe7c4a00acce20e2227b3f355be89bc6827
```

Comment:
Are we really expecting downstream packages to put hashes here ?

Reply:
These hashes are listed automatically on PyPI. Every package uploaded there can be fetched via hash. The neat part is that dependabot will auto-update this file as needed.